observability/opentelemetry/src/main/resources/schema/wildfly-opentelemetry_1_0.xsd

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2021, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           targetNamespace="urn:wildfly:opentelemetry:1.0"
+           xmlns="urn:wildfly:opentelemetry:1.0"
+           elementFormDefault="qualified"
+           version="1.0">
+
+    <xs:element name="subsystem">
+        <xs:complexType>
```

Comment:
Is it ok that configurations for the upcoming parts (like logging) will be mixed up with this one for tracing?

Reply:
An excellent question. Perhaps I should nest all of this under a <tracing> element? @bstansberry, thoughts?

Reply:
We can modify the xsd later if appropriate. There is no requirement to maintain an xml structure across releases, only to be able to parse old versions of the xsd. We make no guarantees that people can do xml manipulation compatibly across releases.

A more important question though is whether the subsystem resource definition structure is properly accounting for likely future expansion. THAT we do provide compatibility guarantees over time. At a glance it seems ok, as the configuration is a number of attributes that are scoped to the 'tracing' use case. New things could be added via child resources or via additional attributes whose names are understandable without clashing with the existing ones.

Reply:
WFM. Thanks.